charts/k8ssandra/values.yaml

```diff
@@ -457,6 +479,17 @@ stargate:
   # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ for background.
   tolerations: []
 reaper:
+  # -- Security context override for reaper container
```

Comment:
Are there security settings that won't work with these containers? If so we should either document it or not allow it. I'm fine with just documenting it and doing that in a follow PR if that works for you.